### ISSUE TITLE
LinkIt is unable to find/add draft Spanish translation nodes

### ISSUE THREAD

**Issue by lburack** [NCIOCPL/cgov-digital-platform#2616]
**What's the issue?**
- Several published English nodes have been translated into Spanish and saved as drafts. The Spanish translations have not been moved through the workflow to a published state.
- I have body text (or a mini landing page content block) that I want to link to an internal link within the CMS. I highlight the text in the WYSIWYG, click the link icon, and start typing the title of the unpublished Spanish page I translated. LinkIt is unable to find the piece of content (see screenshot below)

**User Story/Acceptance Criteria:**
- As a CMS user, I should be able to link to draft content using LinkIt. This applies to content in any language. 
  - This is necessary to allow CMS users to set up new sections of content to publish when ready to go live. This functionality is necessary for creating/setting up newly translated websites. 
- As a CMS user, I should be able to view the unpublished content if I'm logged into the CMS.
- As an anonymous user (not logged into the CMS), unpublished content should not be visible to me (i.e., a list should not include links to unpublished content; a feature card should be empty instead of linking to unpublished content)

![Screen Shot 2020-01-22 at 11 29 57 AM](https://user-images.githubusercontent.com/29731392/72929218-6d993f00-3d27-11ea-9ff3-5f5a04fa3e46.png)

This issue is somewhat related to #2610

**Comment by adriancofie** [comment]
@bennettcc See #2610

**Comment by bennettcc** [comment]
Agreed -- this is categorized as a nice to have but not a blocker for delivery.

**Comment by navsunka** [comment]
The linkit module provides functionality to add an unpublished page to a published page in the ckeditor.
- For logged-in user, it will take us to the unpublished page.
- For anonymous user, It will take us to access denied 403 page.
If we delete the unpublished page that is referred in a published page:
- For all the users, it will take us to page not found 404 page.

**Comment by andyvanavery31** [comment]
Closing this ticket due to the issue being resolved per Naveen's comment above.
